packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -112,6 +112,14 @@ export function resolvePluginConfig(
 		experimental: pluginConfig.experimental ?? {},
 	};
 	const root = userConfig.root ? path.resolve(userConfig.root) : process.cwd();
+	const buildTimeEnv = vite.loadEnv(viteEnv.mode, root, [
```

Comment:
Not sure `buildTimeEnv` is the right name for this. How about `prefixedEnv` or something similar?